### ISSUE TITLE
voc数据集训练未检测到任何目标 Val Loss: nan

### ISSUE THREAD

**Issue by ICE2LAN** [bubbliiiing/yolov4-tiny-pytorch#118]
使用voc数据集训练过程中一直 Val Loss: nan
![image](https://github.com/bubbliiiing/yolov4-tiny-pytorch/assets/69970740/68aa14f5-a366-4992-89de-fce1dee70591)
请问有人遇到过吗，应该怎么处理？谢谢！

**Comment by logic-zhang** [comment]
+1，请问这个问题当时解决了吗

**Comment by logic-zhang** [comment]
> 使用voc数据集训练过程中一直 Val Loss: nan ![image](https://github.com/bubbliiiing/yolov4-tiny-pytorch/assets/69970740/68aa14f5-a366-4992-89de-fce1dee70591) 请问有人遇到过吗，应该怎么处理？谢谢！
+1，请问这个问题当时解决了吗
